src/main/resources/application.properties

```diff
@@ -6,7 +6,9 @@ spring.datasource.password=1234567
 
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.show-sql=true
-spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.show-sql=false
+spring.jpa.properties.hibernate.format_sql=false
+
+logging.config=classpath:logback.xml
```

Comment:
To use simple logs config I think the properties file is enough but since you are using an additional log file configuration considered to add colors to a log levels following the spring documentation [Color-coded Output](https://docs.spring.io/spring-boot/reference/features/logging.html#features.logging.log4j2-extensions.profile-specific:~:text=If%20your%20terminal%20supports)